scss/_layouts_application.scss

```
@@ -89,19 +89,22 @@ $application-layout--side-nav-width-expanded: 15rem !default;
       width: $application-layout--side-nav-width-collapsed;
 
       &.is-collapsed {
+        overflow: hidden;
         transform: translateX(0);
         width: $application-layout--side-nav-width-collapsed;
       }
     }
 
     .l-navigation:hover {
       box-shadow: $panel-drop-shadow;
+      overflow-y: scroll;
```

@sowasred2012 We should use overflow `auto` not to force the scrollbar when it's not needed.

```suggestion
      overflow-y: auto;
```